website/docs/r/iothub_dps.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iot_dps"
+sidebar_current: "docs-azurerm-resource-messaging-iot-dps-x"
+description: |-
+  Manages an IoT Device Provisioning Service.
+---
+
+# azurerm_iot_dps
```

Comment:
I think this file got mislabled? This looks like a copy of [iot_dps.html.markdown](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/r/iot_dps.html.markdown)

Reply:
whoops, i was wondering where these files ended up 🤣